x-pack/plugins/apm/typings/es_schemas/fields/User.ts

```diff
@@ -6,6 +6,6 @@
 
 export interface User {
   id: string;
-  username?: string;
+  user?: string;
```

Comment:
Shouldn't this be `name`?

Reply:
thanks, i had it originally as name but changed back to username to find other ts references, then i slipped up when changing it back...it turns out we don’t even use this property, so we could also delete it if we want.

Reply:
Cool, then I think we should just delete it 👍

Reply:
Perhaps this PR could remove all unused fields?